pages/common/mycli.md

```diff
@@ -0,0 +1,15 @@
+# mycli
+
+> A command line client for MySQL that can do auto-completion and syntax highlighting.
+
+- Connect local database with port 3306 and current user's username:
```

Comment:
```suggestion
- Connect to a local database with port 3306 and the current user's username:
```

Reply:
I'd suggest `on port` rather than `with port`.

Reply:
It sounds a bit odd with `on` for the next part of the sentence. But I agree.

I guess maybe:
> Connect to a local database on port 3306, using the current user's username:

Reply:
@pxgamer that sounds good.